### ISSUE TITLE
Support stable-diffusion-inpainting model

### ISSUE THREAD

**Issue by davidfant** [ShivamShrirao/diffusers#81]
### Model/Pipeline/Scheduler description

RunwayML recently introduced a new model trained specifically for inpainting: https://huggingface.co/runwayml/stable-diffusion-inpainting

The input shape is slightly different:
> For inpainting, the UNet has 5 additional input channels (4 for the encoded masked-image and 1 for the mask itself)

### Open source status

- [X] The model implementation is available
- [X] The model weights are available (Only relevant if addition is not a scheduler).

### Provide useful links for the implementation

Commit where inpainting model was added by @pesser:
https://github.com/runwayml/stable-diffusion/commit/58c459b667069756c3c3bb01e49f1200e3e83d24

**Comment by ShivamShrirao** [comment]
Do u want it supported in dreambooth ?

**Comment by davidfant** [comment]
@ShivamShrirao yes, exactly! If needed, I can e.g. provide a mask of the object I'm trying to make DreamBooth learn
